src/utils/ReduxUtils.ts

```diff
@@ -38,9 +44,12 @@ export interface IReduxAction<T> extends Redux.Action {
 
 export interface IReduxActionPayload {
```

Comment:
It is being used to access all possible payload properties in reducers and mapDispatchToProps.